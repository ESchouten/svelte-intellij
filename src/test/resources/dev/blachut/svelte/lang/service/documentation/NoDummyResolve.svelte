<script lang="ts">
  let unrelatedData = {
    unrelatedValue: 42
  };

  export let usedData: any;

  console.log(unrelatedData);
</script>

{usedData.<caret>unrelatedValue}
