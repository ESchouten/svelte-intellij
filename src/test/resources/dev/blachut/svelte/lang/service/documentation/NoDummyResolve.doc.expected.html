<div class='definition'><pre>unrelatedData.unrelatedValue: <span style="color:#000080;">any</span>
<span style='color:grey;'>[ide type]</span>: <span style="color:#0000ff;">42</span></pre></div><table class='sections'><tr><td valign='top'><icon src='AllIcons.FileTypes.JavaScript'/>&nbsp;NoDummyResolve.svelte</td></table>